### PR TITLE
[BACKLOG-11760] - Localization - Fixes from Acclaro.

### PR DESCRIPTION
**Pull request by dzmitry-hryhoryeu**
[BACKLOG-11760] - Localization - Fixes from Acclaro.

@tmorgner please review.

### CODE REVIEW

**Comment by dzmitry-hryhoryeu** [comment]
@tmorgner please review.

File: package-res/resources/web/dojo/pentaho/common/nls/messages_de.properties [hunk 1]
```diff
@@ -40,3 +40,17 @@ PageControlPreviousPage_title=Vorherige Seite
 #ExpressionTree - labels for filter's operators
 AND=UND
 OR=ODER
+
+# these are for Row Limit Control
+RowLimitLabel=Zeilenlimit:
+RowLimitMaximumTitle= Maximal
+RowLimitNoMoreThanTitle=Nicht mehr als
+UserRowLimitReachedMessage=Sie haben das Zeilenlimit erreicht, Sie k?nnen aber mit dieser Datenteilmenge weiter arbeiten.
```
Comment (tmorgner) [inline package-res/resources/web/dojo/pentaho/common/nls/messages_de.properties]: Same here. Encoding is broken.

**Comment by dzmitry-hryhoryeu** [comment]
@tmorgner please review.

**Comment by dzmitry-hryhoryeu** [comment]
@tmorgner please review.

**Pull request merged**